--- xdaliclock-2.18/X11/digital.c.foo	Tue Nov 30 16:32:40 1999
+++ xdaliclock-2.18/X11/digital.c	Tue Nov 30 16:32:47 1999
@@ -1249,7 +1249,6 @@
             XShapeCombineMask (dpy, window, ShapeBounding,
                                window_offset_x, window_offset_y,
                                pixmap, ShapeSet);
-          else
 #endif /* HAVE_SHAPE */
             /* sends 28 bytes */
             XCopyPlane (dpy, pixmap, window, window_draw_gc, 0, 0,
